### PR TITLE
sub-task/LOC-58/setup-cdk-for-lm-dev-replication

### PR DESCRIPTION
**Pull request by bhuwan-kc**
### Description
 - Replicate LM dev RDS to prod OpenSearch domain using DMS

### CODE REVIEW

File: yarn.lock
Comment (TrevorGentner) [inline yarn.lock:1]: We can delete this right since the project doesn't use yarn?

File: lib/locationDevReplication.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+export class locationDevReplication extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // get location dev RDS secret from secret manager
+    // this secret is a copy of the one in data-dev account
+    // TODO: use data-dev account secret directly (requires cross-account permissions)
+    const secret = sm.Secret.fromSecretAttributes(this, "ImportedSecret", {
+      secretCompleteArn:
+        "arn:aws:secretsmanager:us-east-1:124569967017:secret:locationsDevDB-UlV821"
```
Comment (TrevorGentner) [inline lib/locationDevReplication.ts:17]: Did you just make a copy of the dev db secret?

Another solution could be to create a new user for replication/ ingestion on the dev db that has read perms.

File: yarn.lock
Comment (bhuwan-kc) [reply yarn.lock:1]: I use yarn to install the dependencies. It works a little better than npm itself.

File: lib/locationDevReplication.ts (hunk 1 above)
Comment (bhuwan-kc) [reply lib/locationDevReplication.ts:17]: Yes, I did that as a quick dirty solution. Your proposed solution is much better. I will create a ticket for it.

**Review by TrevorGentner — APPROVED** [review]

**Pull request merged**